content/applications/websites/ecommerce/managing_products/catalog.rst

```diff
@@ -0,0 +1,184 @@
+   You can use **attribute filters** even if you do not work with product variants. When adding
+   attributes to your products, make sure only to specify *one* value per attribute. Odoo will not
+   create variants if no combination is possible.
+
+Sort-by search
+~~~~~~~~~~~~~~
+
+It is possible to allow the user to manually **sort the catalog** using the search bar. From
+:menuselection:`Website dashboard --> Shop`, click on :menuselection:`Edit --> Customize tab`, you
+can enable or disable the :guilabel:`Sort-By` option as well as the :guilabel:`Layout` button. You
+can also select the :guilabel:`Default Sort` of the :guilabel:`Sort-By` button. The default sort
+applies to *all* categories.
+
+The **sorting** options are:
+
+- Featured
+- Newest Arrivals
+- Name (A-Z)
+- Price - Low to High
+- Price - High to Low
+
+In addition, you can **manually edit** the catalog's order of a products by going to
+:menuselection:`Website dashboard --> Shop page --> Edit` and clicking on the product. Under the
+:guilabel:`Product` section of the :guilabel:`Customize` section, you can rearrange the order by
+clicking on the arrows. `<<` `>>` move the product to **extreme** right or left, and `<` `>` move
+the product by **one** row to the right or left. It is also possible to change the catalog's order
+of products in :menuselection:`Website app --> eCommerce --> Products` and dragging-and-dropping the
+products within the list.
+
+.. image:: catalog/catalog-reorder.png
+   :align: center
+   :alt: Product rearrangement in the catalog
+
+Page design
+===========
+
+Category page
+-------------
+
+You can customize the layout of the category page using the website builder. Note that editing the
+layout of the category page is global; editing one category layout will affect *all* category pages.
+
+To do so, go on of your :menuselection:`Category page --> Edit --> Customize`. Here, you can choose
+the layout, the number of columns to display the products, etc. The :guilabel:`Product Description`
+button makes the product description visible from the category page, underneath the product picture.
+
+.. image:: catalog/catalog-category-layout.png
+   :align: center
+   :alt: Layout options of the category pages.
+
+.. tip::
+   You can choose the size of the grid, but be aware that displaying too many products may affect
+   performances and page loading speed.
+
+Product highlight
+-----------------
+
+You can highlight products to make them more visible on the category or product page. On the page of
+your choice, go to :menuselection:`Edit --> Customize` and click on the product to highlight. In the
+:guilabel:`Product` section, you can choose the size of the product image by clicking on the grid,
+and you can also add a **ribbon** or :guilabel:`Badge`. This displays a banner across the product's
+image, such as:
+
+- Sale;
+- Sold out;
+- Out of stock;
+- New.
+
+Alternatively, you can enable the :doc:`Debug mode <../../../general/developer_mode>` on the
```

Comment:
it's named "developer mode", which can be activated ("debug mode" is a remnant of older times)
```suggestion
Alternatively, you can activate the :ref:`developer mode <developer-mode>` on the
```

Reply:
same below

Reply:
Changed